mmrazor/models/algorithms/pruning/dcff.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+import torch
+from mmengine.model import BaseModel
+from mmengine.optim import OptimWrapper
+from torch import nn
+
+from mmrazor.models.distillers import ConfigurableDistiller
+from mmrazor.models.mutables import BaseMutable
+from mmrazor.models.mutators import DCFFChannelMutator
+from mmrazor.models.utils import reinitialize_optim_wrapper_count_status
+from mmrazor.registry import MODELS
+# from mmrazor.structures.subnet.fix_subnet import _dynamic_to_static
+from ..base import BaseAlgorithm
+
+VALID_MUTATOR_TYPE = Union[DCFFChannelMutator, Dict]
+VALID_DISTILLER_TYPE = Union[ConfigurableDistiller, Dict]
+VALID_PATH_TYPE = Union[str, Path]
+VALID_CHANNEL_CFG_PATH_TYPE = Union[VALID_PATH_TYPE, List[VALID_PATH_TYPE]]
+
+
+@MODELS.register_module()
+class DCFF(BaseAlgorithm):
+
+    def __init__(self,
```

Comment:
1. add class docstring for DCFF
2. fuse count -> fuse_freq

Reply:
Updated.